encrypted-media-respec.html

```diff
@@ -3868,23 +3870,48 @@
+            Such tracking would undermine the privacy protections provided by the rest of the web platform, especially those related to isolation of origins.
+          </p>
+          
+          <p>
+            Identifiers, such as <a def-id="distinctive-identifiers"></a>, are often obtained from a server operated or provided by the CDM vendor, such as via an <a href="#individualization">individualization</a> process.
+            The process may include providing client identifier(s), including <a def-id="distinctive-permanent-identifier-maybe-plural"></a>, to the server.
+            In order to generate a per-origin identifier, a value representing the origin may also be provided.
+          </p>
 
+          <p>
+            In such an implementation, the CDM vendor may be able to track the activity of the user, such as number of origins visited or number of times a new identifier is required.
+            If the origin or a value <a def-id="associable"></a> with the origin is provided in the identifier request, the CDM vendor could track the sites visited by the user or user(s) of a device.
+          </p>
           <p>The following section describes techniques that may mitigate the risks of tracking without user consent.</p>
         </section>
         <section>
           <h4>Mitigations</h4>
           <dl>
+            <!-- TODO: Reorder these so related items are near each other. -->
             <dt>Do not <a href="#uses-distinctive-identifiers-or-distinctive-permanent-identifiers">use Distinctive Identifiers or Distinctive Permanent Identifiers</a></dt>
             <dd>
               <p>Key System implementations SHOULD avoid <a href="#uses-distinctive-identifiers-or-distinctive-permanent-identifiers">using Distinctive Identifiers and Distinctive Permanent Identifiers</a> whenever possible and only use them when they meaningfully contribute to the robustness of the implementation.
                 See <a href="#limit-or-avoid-use-of-distinctive-identifiers-and-permanent-identifiers">Limit or Avoid use of Distinctive Identifiers and Permanent Identifiers</a>.
               </p>
             </dd>
 
+            <dt>Use origin-specific and browsing profile-specific Key System storage</dt>
+            <dd>
+              <!-- TODO: Reference section added for issue #242. -->
+              <p>Any data persistently stored by the CDM that might impact messages or behavior in an application- or license server-visible way MUST be stored in an <a def-id="origin"></a>-specific and <a def-id="browsing-profile"></a>-specific way and MUST NOT leak to or from private browsing sessions.
+                Session data, licenses, keys, and other data that are persistently stored MUST be stored per-<a def-id="origin"></a> and per-<a def-id="browsing-profile"></a>.
```

Comment:
This entire section was copied from line 4031 below with one exception - it looks like I added "other data" since this was about tracking in general.

I believe the second sentence is supposed to reinforce the first one. I changed it to note that these are specific examples.